### ISSUE TITLE
AEA PK232 Driver seems to have been broken between versions

### ISSUE THREAD

**Issue by pe1rrr** [pe1rrr/bpq#4]
I've had a report on the group.io about the PK232 sending bunk responses to attach commands, I've set up a PK232 MBX here and confirmed that it does indeed throw a banana.

I am only able to test serial communications as I don't have any radio cables wired for this TNC, however here is the output of my experiment:

This used to work!

```
; Pactor PK232 Port
PORT
        PORTNUM=9
        ID=PK232 Pactor Link
        TYPE=EXTERNAL
        COMPORT=/dev/ttyUSB1
        SPEED=9600;           COM Port Speed
        DRIVER=AEAPactor
        PORTCALL=PE1RRR
        INTERLOCK=9;
;
;
        CONFIG                 ; Driver-Specific Configuration

                MYCALL
                APPL BBS

ENDPORT
```

Attaching to the port results in this:

```
at 9
RIJEN:PE1RRR-7} Ok
c test
AEA} AE
?
AEA} A 
d
AEA} A 
mycall
AEA} AE

d
Disconnected
```

Any assistance with getting sane commands into this would be helpful.

**Comment by ClaudeMa** [comment]
work now on linbpq version 6.0.20.55

**Comment by pe1rrr** [comment]
Issue resolved, upgrade to 6.0.20.55 minimum, requires PK232 firmware 7.2 minimum.
